tbx/work/models.py

```diff
@@ -143,12 +157,131 @@ def type(self):
     ]
 
 
+class WorkPage(SocialFields, Page):
+    template = "patterns/pages/work/work_page.html"
+    parent_page_types = ["WorkIndexPage"]
+
+    intro = RichTextField(blank=True)
+    client = models.CharField(max_length=255, blank=True)
+    date = models.DateField("Post date", blank=True, null=True)
+    body_word_count = models.PositiveIntegerField(null=True, editable=False)
+
+    header_image = models.ForeignKey(
+        "images.CustomImage",
+        verbose_name="Image",
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
+        related_name="+",
+    )
+    header_caption = models.CharField("Caption", max_length=255, blank=True)
```

Comment:
Just to clarify: `verbose_name` values for model fields (and in Model.Meta) should always be lower-case, except for entity names (e.g. Google) and accronyms (e.g. NASA). Django (and by extension, Wagtail) will upper-case the first letter automatically wherever it counts.

Reply:
The above is not true for `label` values on StreamField blocks or form fields, though :)